build_library/disk_util

```diff
@@ -582,7 +582,7 @@ def Update(options):
 
 
 def Mount(options):
-  """Mount the given disk image.
+  """Mount the whole given disk image or (optionally) a single labeled partition from it.
```

Comment:
How about we just create a new command that mounts only the given partition to a specific path, skipping the extra mount directory and bind stuff that only really makes sense for full disk images. To do that split the little bit of common code into a shared function

```python
     mount_opts = ['loop',
                   'offset=%d' % mount['image_first_byte'],
                   'sizelimit=%d' % mount['image_bytes']]
     if options.read_only:
       mount_opts.append('ro')
     elif (mount.get('fs_type', None) in ('ext2', 'ext4') and
           not IsE2fsReadWrite(options, mount)):
       mount_opts.append('ro')
 
     if mount.get('fs_subvolume', None):
       mount_opts.append('subvol=%s' % mount['fs_subvolume'])
 
     Sudo(['mkdir', '-p', full_path])
     Sudo(['mount', '-t', mount.get('fs_type', 'auto'),
                    '-o', ','.join(mount_opts),
                    options.disk_image, full_path])
```